django_app/settings.py

```diff
@@ -12,6 +12,7 @@
 
 import os
 from pathlib import Path
+from decouple import config
```

Comment:
Why do we need this?

Reply:
for getting environment variables from .env file